packages/core/scripts/setup/ios.ts

```diff
@@ -135,74 +140,57 @@ export const addUploadBuildPhase = {
-  );
-  if (nameFounded) {
-    return nameFounded.replace(".xcodeproj", "");
-  }
+export const addEmbraceInitializerSwift = {
+  name: "Adding EmbraceInitializer.swift",
+  run: async (wizard: Wizard): Promise<any> => {
+    const [appId, json] = await wizard.fieldValueList([iosAppID, packageJSON]);
+    const {name} = json;
+    const project = await xcodePatchable({name});
 
-  logger.warn("the xcodeproj file does not match with your project's name");
-  logger.warn(
-    `skipping adding Embrace-Info.plist to ${name}, this should be added to the project manually.
-    You can go https://embrace.io/docs/ios/integration/session-reporting/#import-embrace for more information`,
-  );
-  return name;
-};
+    const filePath = path.join("ios", name, "EmbraceInitializer.swift");
+    if (fs.existsSync(filePath)) {
+      logger.warn("EmbraceInitializer.swift already exists");
+      return;
+    }
 
-export const createEmbracePlist = {
-  name: "Create Embrace plist file",
-  run: (wizard: Wizard): Promise<any> =>
-    wizard.fieldValue(packageJSON).then(({name}) => {
-      const p = path.join("ios", name, "Embrace-Info.plist");
-      if (fs.existsSync(p)) {
-        logger.warn("already has Embrace-Info.json file");
-        return;
-      }
-
-      fs.closeSync(fs.openSync(p, "a"));
-
-      return xcodePatchable({name})
-        .then(project => {
-          const nameWithCaseSensitive = findNameWithCaseSensitiveFromPath(
-            project.path,
-            name,
-          );
-          project.addFile(
-            nameWithCaseSensitive,
-            `${nameWithCaseSensitive}/Embrace-Info.plist`,
-          );
-          project.sync();
-          project.patch();
-        })
-        .then(() => embracePlistPatchable({name}))
-        .then(file =>
-          wizard.fieldValue(iosAppID).then(iosAppIDValue => {
-            file.contents = plistContents(iosAppIDValue);
-            return file.patch();
-          }),
-        );
-    }),
+    fs.writeFileSync(filePath, getEmbraceInitializerContents(appId));
+
+    const nameWithCaseSensitive = findNameWithCaseSensitiveFromPath(
+      project.path,
+      name,
+    );
+    project.addFile(
+      nameWithCaseSensitive,
+      `${nameWithCaseSensitive}/EmbraceInitializer.swift`,
+      "source",
+    );
+    return project.patch();
+  },
   docURL:
     "https://embrace.io/docs/react-native/integration/add-embrace-sdk/#manually",
 };
 
-const plistContents = (iosAppIDValue: string) => {
-  return `<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-<plist version="1.0">
-  <dict>
-    <key>API_KEY</key>
-    <string>${iosAppIDValue}</string>
-    <key>CRASH_REPORT_ENABLED</key>
-    <true/>
-  </dict>
-</plist>`;
+const getEmbraceInitializerContents = (appId: string) => {
+  return `import Foundation
+import EmbraceIO
+
+@objcMembers class EmbraceInitializer: NSObject {
```

Comment:
@ArielDemarco this block shows the contents of the `EmbraceInitializer.swift` file that the install script will automatically add to the app's ios project